res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -680,35 +716,55 @@ PioneerDDJSX.setNonDeckSoftTakeover = function(enable) {
 //                     DECK INIT / RESET                     //
 ///////////////////////////////////////////////////////////////
 
-PioneerDDJSX.initDeck = function(group) {
+PioneerDDJSX.initDeck = function(group, setSoftTakeover) {
     var deck = PioneerDDJSX.channelGroups[group];
 
-    PioneerDDJSX.setDeckSoftTakeover(group, true);
+    if (setSoftTakeover) {
+        PioneerDDJSX.setDeckSoftTakeover(group, true);
+    }
+
     PioneerDDJSX.bindDeckControlConnections(group, true);
 
     PioneerDDJSX.updateParameterStatusLeds(
         group,
         PioneerDDJSX.selectedLoopRollParam[deck],
         PioneerDDJSX.selectedLoopParam[deck],
-        PioneerDDJSX.selectedSamplerBank
+        PioneerDDJSX.selectedSamplerBank,
+        PioneerDDJSX.selectedSlicerQuantizeParam[deck]
     );
     PioneerDDJSX.triggerVinylLed(deck);
     PioneerDDJSX.setDefaultSpeedSliderRange(group, PioneerDDJSX.speedSliderRange);
```

Comment:
I know that but for me it is very comfortable to have a defined slider range with this controller, which also can be changed quickly by pressing a button on the controller. How should I handle changing the speed slider range by the controller button then? Should I consider and save the configured slider range by calling `engine.getValue()` and iterate through my speed slider ranges until it is switched back to the beginning, in this case the configured slider range?